src/components/movie/EditMovieForm.js

```diff
@@ -11,21 +18,32 @@ const EditMovieForm = ({movie}) => {
             },
             {
                 type: 'submit',
-                title: 'Save'
+                title: 'Save',
             }],
         formTitle: 'Edit Movie',
-        action: 'edit',
+        onSubmit: (e) => {
+            props.editMovieRequest(props.movie.id, serializeFormData(e.target));
+            e.preventDefault();
+            return false;
+            },
         descriptions: [
             {text: 'Movie Id', className: 'movie-form__id-title'},
-            {text: movie.id, className: 'movie-form__id-value'}
+            {text: props.movie.id, className: 'movie-form__id-value'}
         ]
     };
 
     return (
         <Form config={editFormConfig}>
-            <MovieFormFields movie={movie}/>
+            <MovieFormFields movie={props.movie}/>
         </Form>
     )
 };
 
-export default EditMovieForm;
+const mapState = (state) => {
+    return {
+        editMovieRequest: state.editMovieRequest,
```

Comment:
if it's a function it should be moved to `mapDispatchToProps` and should NOT be a part of store data

Reply:
Fixed